app/modules/organizations/models.py

```diff
@@ -5,28 +5,145 @@
 """
 
 from app.extensions import db, HoustonModel
+from app.extensions.edm import EDMObjectMixin
 
+import logging
 import uuid
+import datetime
+import pytz
 
+# todo, this should be in config.py, not across various files in the system, and yes should be called TIMEZONE, not PST
+TIMEZONE = pytz.timezone('US/Pacific')
+DATETIME_FMTSTR = '%Y-%m-%dT%H:%M:%S.%fZ'
 
-class Organization(db.Model, HoustonModel):
+log = logging.getLogger(__name__)
+
+
+class OrganizationEDMMixin(EDMObjectMixin):
+
+    # fmt: off
+    # Name of the module, used for knowing what to sync i.e organization.list, organization.data
+    EDM_NAME = 'organization'
+
+    # The EDM attribute for the version, if reported
+    EDM_VERSION_ATTRIBUTE = 'version'
+
+    #
+    EDM_LOG_ATTRIBUTES = [
+        'name',
+    ]
+
+    # todo Very much a first drop of mapping
+    EDM_ATTRIBUTE_MAPPING = {
+        # Ignored
+        'id'                    : None,
+        'created'               : None,
+        'modified'              : None,
+
+        # # Attributes
+        'name'                  : 'title',
+        'url'                   : 'website',
+        'version'               : 'version',
+
+        # # Functions
+        'members'               : '_process_members',
+        'logo'                  : '_process_logo',
+        'createdDate'           : '_process_created_date',
+        'modifiedDate'          : '_process_modified_date',
+    }
+    # fmt: on
+
+    @classmethod
+    def ensure_edm_obj(cls, guid):
+        organization = Organization.query.filter(Organization.guid == guid).first()
+        is_new = False
+
+        if organization is None:
+            organization = Organization(
+                guid=guid,
+                title='none',
+            )
+            db.session.add(organization)
+            is_new = True
+
+        return organization, is_new
+
+    def _process_members(self, members):
+        from app.modules.users.models import User
+
+        for member in members:
+            log.info('Adding Member ID %s' % (member.id,))
+            user, is_new = User.ensure_edm_obj(member.id)
+
+            # todo restore once the tests work
```

Comment:
The pedant in me says this line should be removed but the realist says this needs to be pulled without any further delay.

Reply:
Removed!